### ISSUE TITLE
Bug - Orbat list unit icons inconsistencies

### ISSUE THREAD

**Issue by Sniperhid** [alexcroox/R3#17]
I was watching one of the replays on the FA server and noticed some quirks - http://server.folkarps.com/r3/share/10

It is possible for unit icons to disappear for dead units if you jump around (35:28) using the timeline bar, which then looks like this:
https://gyazo.com/f4c247ab1af94ef94169b3a59e4300a0

However if you watch the playback from beginning to end without clicking (jumping) on the timeline there will never be missing/empty icons for units.

Another interesting quirk I noticed is that if you jump to (49:14). Suddenly all units appear in vehicles including the dead ones.
https://gyazo.com/38b008a122ac220e34fd22c1304c62bb

I imagine consistent behaviour would be nice. Perhaps the best solution here is adding a dead icon for units that are dead - as this would make it extremely clear who is alive in the platoon at any given time?

**Comment by alexcroox** [comment]
Agreed. With the rewrite that's underway the addon and extension now store data in a totally different way which I haven't even proven will work in the UI yet, however I will make sure there is a focus on keeping the player side bar as accurate as possible throughout playback. It currently works off get in and out events which of course don't always fire if you skip through time so perhaps an occasional loop to validate the state of the sidebar unit icons would help
